src/components/interactive-builder/add-question-modal.component.tsx

```diff
@@ -675,6 +709,80 @@ const AddQuestionModal: React.FC<AddQuestionModalProps> = ({
                   />
                 </RadioButtonGroup>
               ) : null}
+
+              {questionType === 'programState' && (
+                <Stack gap={5}>
+                  <div>
+                    {isLoadingPrograms && <SelectSkeleton />}
+                    {programsLookupError ? (
+                      <InlineNotification
+                        kind="error"
+                        lowContrast
+                        className={styles.error}
+                        title={t('errorFetchingPrograms', 'Error fetching programs')}
+                        subtitle={t('pleaseTryAgain', 'Please try again.')}
+                      />
+                    ) : null}
+                    {programs && (
+                      <ComboBox
+                        id="programLookup"
+                        items={programs}
+                        itemToString={(item: Program) => item?.name}
+                        onChange={({ selectedItem }: { selectedItem: Program }) => {
+                          handleProgramChange(selectedItem);
+                        }}
+                        placeholder={t('addProgram', 'Add program')}
+                        selectedItem={selectedProgram}
+                        titleText={t('program', 'Program')}
+                      />
+                    )}
+
+                    {selectedProgram && (
+                      <ComboBox
+                        id="programWorkflowLookup"
+                        items={programWorkflows}
+                        itemToString={(item: ProgramWorkflow) => item?.concept?.display}
+                        onChange={({ selectedItem }: { selectedItem: ProgramWorkflow }) =>
+                          handleProgramWorkflowChange(selectedItem)
+                        }
+                        placeholder={t('addProgramWorkflow', 'Add program workflow')}
+                        selectedItem={programWorkflowUuid}
```

Comment:
```suggestion
                        selectedItem={programWorkflows.find((workflow) => workflow.uuid === programWorkflowUuid)}
```

Reply:
I think the variable name is confusing, i will change it to programWorkflow, a user should chose a workflow from the list, this change is handled on edit view